### PR TITLE
[PE-19] Update two of the RARE-X landing page hyperlinks

### PR DESCRIPTION
**Pull request by kyuksel**
# Ticket
https://broadworkbench.atlassian.net/browse/PE-19

# Before

image

# After

image

### CODE REVIEW

**Review by nawatts — APPROVED** [review]
> I'll update the base as dev once https://github.com/DataBiosphere/terra-ui/pull/3437 is merged.

It doesn't look like there's anything in here that depends on that PR.

**Comment by kyuksel** [comment]
> > I'll update the base as dev once #3437 is merged.
> 
> It doesn't look like there's anything in here that depends on that PR.

There isn't. I happened to branch off of that PR's branch and figured I'd keep it to save myself some merge conflicts. Sorry if that was confusing.

File: src/libs/brands.js [hunk 1]
```diff
@@ -198,12 +198,12 @@ export const brands = {
     hostName: 'rare-x.terra.bio',
     docLinks: [
       {
-        link: 'https://rare-x.org/DataAnalysisPlatform-Documentation',
-        text: 'Find RARE-X Data Analysis Platform documentation, tutorials and Jupyter notebook examples'
+        link: 'https://docs.google.com/forms/d/e/1FAIpQLScpqfZIXW53IJOiYz2RhASJzm7lZBYAjzkjJ67qFDERUpuDAQ/viewform',
```
Comment (dmohs) [inline src/libs/brands.js:201]: FYI: This doc requires Google authentication.

**Review by dmohs — APPROVED** [review]

File: src/libs/brands.js (hunk 1 above)
Comment (kyuksel) [reply src/libs/brands.js:201]: Correct. Do we do anything differently on the UI when that link requires authentication?

**Pull request merged**